presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries that are submitted when the cluster does not have the minimum required number of workers available.
+* Fix an issue causing a union of a table with a ``VALUES`` statement to execute on a single machine, which may result in out of memory errors.
+* Improve the logical plan by removing unnecessary remote exchanges before window functions when the data is already available locally.
+* Add grouped execution support for window functions.
+* Add support for the ``BOOLEAN`` type for ``EXPLAIN IO`` statements.
```

Comment:
This seemed like maybe it was just a bug before?

Reply:
No it was not. We deliberately limited the supported types (see discussions here: https://github.com/prestodb/presto/pull/11049#discussion_r206356547)